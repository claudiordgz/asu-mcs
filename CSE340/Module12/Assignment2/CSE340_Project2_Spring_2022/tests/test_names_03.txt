a, b, c;

test {
    public:
        a, b, hello;
    private:
        x, y;
    a = b;
    hello = c;
    y = r;
    nested {
        public:
            b;
        a = b;
        x = hello;
        c = y;
    }
}