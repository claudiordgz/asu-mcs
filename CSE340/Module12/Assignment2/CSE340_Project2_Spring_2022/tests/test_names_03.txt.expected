test.a = test.b
test.hello = ::c
test.y = ?.r
test.a = nested.b
?.x = test.hello
::c = ?.y